src/client/lazy-app/Compress/Output/custom-els/TwoUp/index.ts

```diff
@@ -84,20 +77,36 @@ export default class TwoUp extends HTMLElement {
         }</svg>
       `}</div>`;
 
+    // Watch for element size changes.
+    if ('ResizeObserver' in window) {
+      this._resizeObserver = new ResizeObserver(() => this._resetPosition());
+      this._resizeObserver.observe(this);
+    } else {
+      window.addEventListener('resize', this._onResize);
+    }
+
+    window.addEventListener('keydown', this._onKeyDown);
+
     if (!this._everConnected) {
       this._resetPosition();
       this._everConnected = true;
     }
   }
 
+  disconnectedCallback() {
+    window.removeEventListener('keydown', this._onKeyDown);
+    window.removeEventListener('resize', this._onResize);
+    if (this._resizeObserver) this._resizeObserver.disconnect();
```

Comment:
I really don't think we should have to do this, but it's a direct cause of the leak. Filed https://bugs.chromium.org/p/chromium/issues/detail?id=1219954.